### DISCUSSION TITLE
where is Elastic License

### DISCUSSION THREAD

**Discussion by Ed1s0nZ** [Security-Onion-Solutions/securityonion#4830] [Unsupported Versions] [answered]
![image](https://user-images.githubusercontent.com/83812544/125886587-30573062-0e14-4219-8f27-76139ee611e1.png)
What should I do next?
I didn't find an Elastic License.
help me

**Comment by pkgwillie** [comment, marked as answer]
You need to type AGREE in the box to proceed.
